georocket-server/src/main/java/io/georocket/index/elasticsearch/RemoteElasticsearchClient.java

```diff
@@ -191,7 +191,35 @@ public void close() {
       }
     });
   }
-  
+
+  @Override
+  public Observable<Boolean> putSettings(JsonObject settings) {
+    String indexUri = "/" + index;
+    String settingsUri = indexUri + "/_settings";
+    String openUri = indexUri + "/_open";
+    String closeUri = indexUri + "/_close";
+
+    // Elasticsearch need to close a index befor it can be updated.
+    return performRequestRetry(HttpMethod.POST, closeUri, null)
```

Comment:
You only need to close the index if you want to add analyzers. Updating the settings without analyzers is a valid operation that should not require the index to be closed and opened again.